internalstaging.theanvil.io/manifest.json

```diff
@@ -7,28 +7,28 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "access-backend": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/access-backend:1.1",
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2020.10",
+    "access-backend": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/access-backend:2020.11",
```

Comment:
Do we have a `2020.11` tag for access-backend? Looks like we don't: https://github.com/uc-cdis/ACCESS-backend/tags

Reply:
Goog catch. Pushing a commit to fix it...